doc/source/serve/getting_started.md

```diff
@@ -2,9 +2,20 @@
 
 # Getting Started
```

Comment:
Can we have a table of contents in this page? Lets user target their interests quickly.